CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 ### Internals
 * Updates for upcoming Platform Networking feature, including new SyncSocketProvider class. ([PR #6096](https://github.com/realm/realm-core/pull/6096))
 * Updated namespaces for files moved to realm/sync/network ([PR #6109](https://github.com/realm/realm-core/pull/6109))
+* Create the fuzzer framework project in order to run fuzz testing on evergreen.
```

Comment:
There does not appear to be anything running on evergreen?

Reply:
@fealebenpae helped me to fix the compilation issues, going to add the logic to the evergreen configuration file in order to run it.